src/main/java/org/example/EmployeeBook.java

```diff
@@ -0,0 +1,382 @@
+package org.example;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class EmployeeBook {
+    private Employee[] employees;
+
+    private int length = 10;
+
+    public EmployeeBook(int numberOfWorkspaces) {
+        employees = new Employee[length];
```

Comment:
Наверное имел ввиду так
```suggestion
        employees = new Employee[numerOfWorkspaces];
```

Reply:
Не сошлись названия переменных...
должно быть так
`        employees = new Employee[numberOfWorkspaces];
        length = numberOfWorkspaces;`
        
length  являет собой возможную длинну массива сотрудников.
numberOfWorkspaces являет собой то же самое.
По хорошему стоит переименовать numberOfWorkspaces  в  length